<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini Text</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_Neu"/>
    <addaction name="action_ffnen"/>
    <addaction name="action_Speichern"/>
    <addaction name="actionSpeichern_unter"/>
    <addaction name="separator"/>
    <addaction name="action_Beenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menu_Datei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Neu"/>
   <addaction name="action_ffnen"/>
   <addaction name="action_Speichern"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Neu">
   <property name="icon">
    <iconset resource="symbole.qrc">
     <normaloff>:/symbol/dateiNeu.bmp</normaloff>:/symbol/dateiNeu.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Neu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="symbole.qrc">
     <normaloff>:/symbol/dateiOeffnen.bmp</normaloff>:/symbol/dateiOeffnen.bmp</iconset>
   </property>
   <property name="text">
    <string>Ö&amp;ffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Speichern">
   <property name="icon">
    <iconset resource="symbole.qrc">
     <normaloff>:/symbol/dateiSpeichern.bmp</normaloff>:/symbol/dateiSpeichern.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Beenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="symbole.qrc">
     <normaloff>:/symbol/info.bmp</normaloff>:/symbol/info.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="icon">
    <iconset resource="symbole.qrc">
     <normaloff>:/symbol/dateiSpeichern.bmp</normaloff>:/symbol/dateiSpeichern.bmp</iconset>
   </property>
   <property name="text">
    <string>Speichern unter…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="symbole.qrc"/>
 </resources>
 <connections/>
</ui>
